server/config.go

```diff
@@ -477,8 +478,29 @@ func (o *scheduleOption) RemoveSchedulerCfg(name string) error {
 	return nil
 }
 
-func (o *scheduleOption) persist(kv *kv) error {
-	return kv.saveScheduleOption(o)
+func (o *scheduleOption) persist(kv *core.KV) error {
+	cfg := &Config{
+		Schedule:    *o.load(),
+		Replication: *o.rep.load(),
+	}
+	err := kv.SaveConfig(cfg)
+	return errors.Trace(err)
+}
+
+func (o *scheduleOption) reload(kv *core.KV) error {
```

Comment:
`reload` is previously a method of server. see https://github.com/pingcap/pd/pull/781/files#diff-2c671b5ee7007449d6115af14481769eL370